core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -83,7 +83,7 @@
     private boolean redistributeWrites = true;
     private boolean usePreferredWritePartitioning = true;
     private int preferredWritePartitioningMinNumberOfPartitions = 50;
-    private boolean scaleWriters;
+    private boolean scaleWriters = true;
```

Comment:
This should come after the commit adding REPORT_BYTES_WRITTEN. reorder the commits

Reply:
Done